navigation.yaml

```diff
@@ -495,171 +496,167 @@ products:
               url: https://juju.is/docs/sdk/from-zero-to-hero-write-your-first-kubernetes-charm
 
 solutions:
-  - cta: Explore our solutions
-    main_lists:
-      - title: Categories
-        primary_links:
-          - links:
-              - title: AI
-                url: /solutions/ai
-              - title: Data
-                url: /data
-              - title: Infrastructure
-                url: /solutions/infrastructure
-              - title: IoT
-                url: /PLACE-HOLDER
-              - title: Managed IT services
-                url: /PLACE-HOLDER
-              - title: Secure open source
-                url: /PLACE-HOLDER
-              - title: Ubuntu OS
-                url: /PLACE-HOLDER
+  cta: Explore our solutions
+  main_lists:
+    - title: Categories
+      link_list:
+        - title: AI
+          url: /solutions/ai
+        - title: Data
+          url: /data
+        - title: Infrastructure
+          url: /solutions/infrastructure
+        - title: IoT
+          url: /PLACE-HOLDER
```

Comment:
Are the placeholder URLs supposed to be updated in this PR?

Reply:
No, these will come at a later date